.travis.yml

```diff
@@ -1,83 +0,0 @@
-language: python
-
-sudo: false
-
-dist: xenial
-
-env:
-  global:
-  - OPENBLAS_NUM_THREADS=1
-  - OMP_NUM_THREADS=1
-  - MKL_NUM_THREADS=1
-  # Directory where tests are run from
-  - TEST_DIR=/tmp/test_dir/
-  - EXAMP_DIR=/tmp/examp_dir/
-  - MODULE=autosklearn
-  - MINICONDA_URL="https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh"
-
-matrix:
-  allow_failures:
-  - os: osx
-
-  include:
-  - os: linux
-    env: DISTRIB="conda" DOCPUSH="true" PYTHON="3.7" SKIP_TESTS="true"
```

Comment:
The new tests do no longer test whether we can install in a conda environment. Could you please re-add those tests?